plugins/flytekit-k8sdataservice/k8s_ops/k8s-service-agent-role.yaml

```diff
@@ -0,0 +1,86 @@
+# Example of the role/binding set up for the data service to create/update/delete resources in the sandbox flyte namespace
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Instead of adding this file to flytekit, could we add this to the agent setup guide? https://docs.flyte.org/en/latest/deployment/agents/databricks.html#deployment-agent-setup-databricks